emiproc/__init__.py

```diff
@@ -231,10 +231,15 @@ def process_tno(cfg, interpolation, country_mask, out, latname, lonname):
                 ):
                     if sel.any():
                         out_var_name = util.get_out_varname(s, cat, cfg,
-                                                       source_type=t)
+                            				source_type=t)
                         print('Write as variable:', out_var_name)
-                        util.write_variable(out, out_var, out_var_name,
-                                            latname, lonname, unit)
+                        if cfg.model.startswith("cosmo"):
+                            util.write_variable(out, out_var, out_var_name,
+                                                latname, lonname, unit)
+                        elif cfg.model.startswith("icon"):
+                            print(out_var.shape)
```

Comment:
You might want to delete this `print(out_var.shape)`, which was probably added for debugging?